packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -231,6 +231,60 @@ class _ProductListPageState extends State<ProductListPage>
         actionModeTitle: Text(appLocalizations.compare_products_appbar_title),
         actionModeSubTitle:
             Text(appLocalizations.compare_products_appbar_subtitle),
+        actionModeActions: <Widget>[
+          IconButton(
+            icon: const Icon(Icons.delete),
+            onPressed: () async {
+              if (_selectedBarcodes.isNotEmpty) {
+                await showDialog<bool>(
```

Comment:
You say that your dialog returns a boolean, but all your `Navigator.pop` returns nothing.
There is a problem here.

Reply:
I guess it would be better to make it `void` since we are not returning any value here? what say @g123k

Reply:
If you don't use the value, then it's ok to do this instead